pkg/edgecache/cache_volume/pvutil.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cachevolume
```

Comment:
pvutil.go seems to be in 2 places

Reply:
its duplicated since the cache volume code will be ripped out, but `nodeserver.go` uses those functions as well

i could remove the duplication and let marisa do it if thats preferred (to make the PR more clear)

Reply:
duplicated the tests